charts/operator/README.md

```diff
@@ -116,15 +122,24 @@ matchers:
             value: "<YOUR TOKEN>"
 ``` 
 
-## Check deployment status
+## Important log lines
 
-Get all deployment logs :
+How to see the operator logs ?
 ```
 kubectl -n rookout logs -f deployment/rookout-controller-manager --all-containers=true --since=10m
 ```
 
-You should see the following log message :
+Indicates that the operator is ready for patching deployments :
 ```
 Operator configuration updated
 ```
+If the above log line not found it means no operator matchers supplied. 
+make sure at least one matcher defined in [values.yaml](./values.yaml) under the operator.matchers section.
```

Comment:
Make sure at least one matcher is defined in [values.yaml](./values.yaml) under the operator.matchers section.